src/hooks/useUsers.tsx

```diff
@@ -10,15 +11,15 @@ const useUsers = () => {
   const { axiosManualOptions } = useMyAxios()
   const [, execute] = useAxios({}, axiosManualOptions)
 
-  const getUsers = (): AxiosPromise<{ users: User[] }> => {
+  const getUsers = useCallback((): AxiosPromise<{ users: User[] }> => {
```

Comment:
This refactor is part of an effort to eliminate/reduce warnings from unit tests for `Routes.tsx`.